Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -2581,10 +2581,14 @@ class CommandResults:
     :return: None
     :rtype: ``None``
     """
-    def __init__(self, outputs_prefix, outputs_key_field, outputs, indicators=None, readable_output=None,
+    def __init__(self, outputs_prefix='', outputs_key_field='', outputs=None, indicators=None, readable_output=None,
```

Comment:
```suggestion
    def __init__(self, outputs_prefix=None, outputs_key_field=None, outputs=None, indicators=None, readable_output=None,
```

Reply:
string isn't a mutable object, so it's fine to use empty strings.